examples/jsm/renderers/common/Animation.js

```diff
@@ -1,44 +1,39 @@
 class Animation {
 
-	constructor() {
+	constructor( nodes, info ) {
 
-		this.nodes = null;
+		this.nodes = nodes;
+		this.info = info;
 
 		this.animationLoop = null;
 		this.requestId = null;
 
-		this.isAnimating = false;
-
-		this.context = self;
+		this._init();
 
 	}
 
-	start() {
-
-		if ( this.isAnimating === true || this.animationLoop === null || this.nodes === null ) return;
-
-		this.isAnimating = true;
+	_init() {
 
 		const update = ( time, frame ) => {
 
 			this.requestId = self.requestAnimationFrame( update );
```

Comment:
Actually... is this correct that this function is called every frame using rAF even if `animationLoop ` isn't set? /ping @sunag

Reply:
It was intentional, `nodeFrame.update()` cannot be called in `render()` to work correctly, this fix cases where no `animationLoop` was added.

Reply:
We need to know what frame we are in to trigger `NodeUpdateType.FRAME` frame events, otherwise we will only have `NodeUpdateType.RENDER` call events. The frame counter makes sure which frame is in order to avoid duplicate events.